lib/elixir/test/elixir/registry_test.exs

```diff
@@ -231,6 +266,17 @@ defmodule RegistryTest do
         assert length(Supervisor.which_children(registry)) == partitions
       end
 
+      test "counts 0 keys in an empty registry", %{registry: registry} do
+        assert 0 == Registry.count(registry)
+      end
+
+      test "counts the number of keys in a registry", %{registry: registry} do
+        {:ok, _} = Registry.register(registry, "hello", :value)
+        {:ok, _} = Registry.register(registry, "hello", :value)
+
+        assert 2 == Registry.count(registry)
+      end
+
```

Comment:
This set ot count tests is duplicated

Reply:
I've duplicated to test for both `:unique` and `:duplicate` registries